pokemon-showdown

```diff
@@ -2,14 +2,16 @@
 'use strict';
 
 const child_process = require('child_process');
+const fs = require('fs');
+const path = require('path');
 
 // Make sure we're Node 6+
 
 try {
 	eval('{ let [a] = [1]; }');
 } catch (e) {
 	console.log("We require Node.js v6 or later; you're using " + process.version);
-	process.exit();
+	process.exit(0);
```

Comment:
[So following Node's conventions for error codes](https://nodejs.org/api/process.html#process_exit_codes), it should be 128 + 69